README.md

```diff
@@ -484,6 +484,113 @@ This file will be pushed via a **separate `link` element in the `head`,** prior
 
 This file currently **WILL NOT** be compiled with LESS, and it **MAY NOT** set LESS variables for other stylesheets to honor. Again, its primary purpose is to declare font face imports in a way that does not require excessive imports that are not needed in other locales.
 
+### Parked pages with localized slugs
+
+In `apostrophe-pages` configuration, you can have parked pages created automatically on server startup. The usual way is to configure a `slug` string for each parked page. If you need localized slugs, there will be a `slug` object instead:
+
+```javascript
+park: [
+  {
+    slug: '/',
+    published: true,
+    _defaults: {
+      title: 'Home',
+      type: 'home'
+    },
+    _children: [
+      {
+        slug: {
+         'en': '/products-en',
+         'fr': '/produits',
+         '_default': '/products'
+        },
+        _defaults: {
+          type: 'product-page',
+          title: 'Product'
+        },
+        published: true,
+        parkedId: 'products'
+      },
+    ]
+  }
+]
+```
+
+The slug keys must match workflow locales. Note the `_default` property for any locales not enumerated in the `slug` object. It is mandatory if at least one locale from `apostrophe-workflow` is not mentioned.
+Also, the homepage (`'/'`) cannot be localized this way because prefixes can be configured in `apostrophe-workflow`.
+
+#### Prefixes in localized slugs
+
+If `apostrophe-workflow` has prefixes, it will be automatically to the slugs defined in `apostrophe-pages` parked pages.
+
+For example:
+
+```javascript
+// app.js
+require('apostrophe')({
+  shortName: 'test',
+  modules: {
+    'apostrophe-workflow': {
+      prefixes: {
+        'en': '/en',
+        'fr': '/fr',
+        'de': '/de'
+      }
+    },
+    'apostrophe-pages': {
+      types: [
+        {
+          name: 'home',
+          label: 'Home'
+        },
+        {
+          name: 'product-page',
+          label: 'Product'
+        }
+      ],
+      park: [
+        {
+          slug: '/',
+          published: true,
+          _defaults: {
+            title: 'Home',
+            type: 'home'
+          },
+          _children: [
+            {
+              slug: {
+              'en': '/products-en',
+              'fr': '/produits',
+              '_default': '/products'
+              },
+              _defaults: {
+                type: 'product-page',
+                title: 'Product'
+              },
+              published: true,
+              parkedId: 'products'
+            }
+          ]
+        }
+      ]
+    },
+    product: {
+      extend: 'apostrophe-pieces',
+      name: 'product',
+      label: 'Product'
+    },
+    'product-pages': {
+      extend: 'apostrophe-pieces-pages'
+    }
+  }
+});
+```
+
+With this configuration, the product page URLs created at server startup will be :
+- for the `en` locale: `en/products-en`
```

Comment:
These examples should include the leading `/`

Reply:
Ok